builtin/providers/google/resource_compute_instance_group_test.go

```diff
@@ -201,7 +211,34 @@ func testAccComputeInstanceGroup_named_ports(n string, np map[string]int64, inst
 	}
 }
 
-func testAccComputeInstanceGroup_basic(instance string) string {
+func testAccComputeInstanceGroup_network(n string, network string, instanceGroup *compute.InstanceGroup) resource.TestCheckFunc {
```

Comment:
nitpick: can you rename this to something like `testAccComputeInstanceGroup_networkExists`? Check functions (in my opinion) should be more boolean-like in name to make it clear that it's a check and not a config.